src/App.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+ <div>
+   <h1>ToDoリスト</h1>
+   <label><input type="radio" name="task" @click="display = 'すべて'" checked>すべて</label>
+   <label><input type="radio" name="task" @click="display = '作業中'">作業中</label>
+   <label><input type="radio" name="task" @click="display = '完了'">完了</label>
+
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(todo,index) in Todos" v-bind:key="index" v-show="display===todo.status || display==='すべて'">
+            <th>{{ index }}</th>
+            <th>{{ todo.task }}</th>
+            <th><button @click="changeStatus(index)">{{ todo.status }}</button></th>
+            <th><button @click="deleteTask(index)">削除</button></th>
+        </tr>
+      </tbody>
+    </table>
+   <h2>新規タスクの追加</h2>
+   <input type="text" v-model="task"><button @click="pushTask">追加</button>
+
+ </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      task: '',
+      status: '作業中',
+      display: 'すべて',
+      Todos: []
```

Comment:
JSではローワーキャメルケースを採用するのが通例ですので、
下記のように修正しましょう。
```suggestion
      todos: []
```